pattern_manager/PatternParent parent
uint64 num_points
float64 step_size
float64 length
---
bool success